lib/src/json_schema/json_schema.dart

```diff
@@ -1040,19 +1060,20 @@ class JsonSchema {
       // Note: see https://json-schema.org/draft/2020-12/release-notes.html
 
       // Added or changed in draft2020_12
-      r'prefixItems': (JsonSchema s, dynamic v) => s._setPrefixItems(v), // TODO: implement
-      r'items': (JsonSchema s, dynamic v) => s._setItemsDraft2020(v), // TODO: Change implementation
-
-      r'$dynamicRef': (JsonSchema s, dynamic v) => null, //TODO: implement
-      r'$dynamicAnchor': (JsonSchema s, dynamic v) => null, //TODO: implement
+      r'prefixItems': (JsonSchema s, dynamic v) => s._setPrefixItems(v),
+      r'items': (JsonSchema s, dynamic v) => s._setItemsDraft2020(v),
 
-      // r'contains': (JsonSchema s, dynamic v) => null, // TODO: Change implementation
-      // r'unevaluatedItems': (JsonSchema s, dynamic v) => null // TODO: Change implementation
+      r'$dynamicRef': (JsonSchema s, dynamic v) => s._setDynamicRef(v),
+      r'$dynamicAnchor': (JsonSchema s, dynamic v) => s._setDynamicAnchor(v),
     });
 
   /// Get a nested [JsonSchema] from a path.
   JsonSchema resolvePath(Uri path) => _getSchemaFromPath(path);
 
+  /// Get a [JsonSchema] from the dynamicParent with the given anchor. Returns null if none exists.
+  JsonSchema resolveDynamicAnchor(JsonSchema dynamicParent, String anchor) =>
```

Comment:
should this be called `dynamicRef` from the consumer's context?
```suggestion
  JsonSchema resolveDynamicAnchor(JsonSchema dynamicRef, String dynamicAnchor) =>
```

Reply:
I think it should be `dynamicParent` from the consumers context. Because this function expects the a schema from the dynamic parent lineage to be passed in. Does that even make sense?

 I think `dynamicAnchor` is a good change.

Reply:
I guess that other question is, what's the use-case for this? Does the user ever care to resolve things this way? Or would they just as well use the resolvePath method?

Reply:
If we don't have a strong need for this public surface I'd just omit it for now.

Reply:
Oh I see you're using this in the validator. Is this more logic that should just be private in the validator?

Reply:
The best strategy was discussed and decided upon in chat.